internal/provider/resource_apply_module.go

```diff
@@ -191,6 +198,47 @@ func (t *applyModuleResource) Schema(_ context.Context, _ resource.SchemaRequest
 					},
 				},
 			},
+			"speculative": schema.BoolAttribute{
+				MarkdownDescription: "Whether the run will be speculative, default is false.",
+				Description:         "Whether the run will be speculative, default is false.",
+				Optional:            true,
+				Default:             booldefault.StaticBool(false),
+				Computed:            true, // Must be computed if setting a default in the schema.
```

Comment:
This shouldn't need to be computed since the default value will be used when it's not provided.

Reply:
Without the Default field, it did not need to also be Computed.  However, if Default is set, Terraform refuses to accept the provider if Computed is not set.  Do you need me to rerun the experiment and send a screenshot?

Reply:
I reran the experiment.  Based on the error message, Terraform refuses to accept a default field without computed.  Please see https://tharsis.dev.cts.infor.com/groups/rmr/g616a/bp3/primary/ws1/-/runs/Ul81ZmJiNTlmZC1kM2IyLTRiNjMtOGQ3My0wMzg5YzRkNWRkNWI/plan  This is the terminal's view:
![image](https://github.com/martian-cloud/terraform-provider-tharsis/assets/86491630/718272d4-8e6b-4347-9fbc-31fc7919ad77)